### DISCUSSION TITLE
Accidentally clicked on "Unsubscribe" on the "Notifications" inbox

### DISCUSSION THREAD

**Discussion by ovflowd** [community/community#11732] [Other Feature Feedback, Questions, & Ideas]
### The Issue

The title is quite straightforward. Sometimes when I want to click on "Done" on some of my subscribed topics, I accidentally click on "Unsubscribe". Which (per the GitHub Docs) will prevent further notifications from those topics on my Notifications Inbox.

I searched through the whole GitHub Docs but haven't seen any option to undo, or re-subscribe to the Notifications that I've accidentally unsubscribed.

### The Idea

When you click on "Done", "Unsubscribe" or any other action, tho show an alert/toast for an X amount of seconds which allows you to undo the action.

### What bakes this idea?

Many platforms such as, for example, but not limited by, Google Mail, Facebook and others allow you to undo actions that might have been done accidentally for a short period of time.

### Alternate Solution

Allow somewhere else, or inside the Notifications Inbox to see a full list of the topics/notifications you've Unsubscribed from your Inbox, and allow to re-subscribe to them. (This might not be necessary for actions such as "Done")

**Comment by paramsiddharth** [comment]
It just happened with me too and now I am annoyed too. We definitely need to have this fixed.

P.S. Hey Claudio @ovflowd ! I'm Param from the GitHub server. :)

**Reply by ovflowd** [reply]
Oh hey there 👋

**Comment by jfullstackdev** [comment]
if I accidentally click `Unsubscribe`

![image](https://user-images.githubusercontent.com/47092464/182910570-5212fc20-ddc1-4ed1-a047-3812349e75cd.png)

then, they are still at the `Done` tab

![image](https://user-images.githubusercontent.com/47092464/182910658-f4be4187-2422-43c2-9c30-7504e151f2c1.png)

you just need to click that Subscribe button again

**Reply by paramsiddharth** [reply]
Wow, I never realized that was there. Thank you, @jdevstatic !

**Reply by whyprax** [reply]
Thank you, man. I was so stressed out.

**Reply by martin-braun** [reply]
@jdevstatic Bless you, you just saved my day. :)

**Reply by Hammerfest** [reply]
this thought is great and all, but I did this long ago and it was when I was trying to figure out how to clear the notifications when I hit that unsubscribe option, so nothing ever went to the done section

Github really needs to have an "heres all the stuff you unsubscribed from" list

**Comment by TallTed** [comment]
Similar, but different — I ticked the checkbox on a half-dozen notifications, and accidentally clicked the `Done` button up top, when I meant to set them back to `Unread`. (They were showing `Read` because of some other bug in GitHub, which *mostly* doesn't bother me, except when I have a brain-glitch like this one.) Is there any hope of an UNDO or other recovery of those notifications?  (I should have them all in my email, but there are thousands of messages there, and no easy way to discern which I've accidentally `Done`.)

**Reply by TallTed** [reply]
I finally found what was referred to above as the "`Done` tab". There's no TAB for it; it's in the upper-left nav sub-menu, with `Inbox` and `Saved`, above the `Filters` and `Repositories` sub-menus. I don't understand why people haven't also shared [the direct link](https://github.com/notifications?query=is%3Adone)... but there, now I have.
